src/App.js

```diff
@@ -1,41 +1,43 @@
 import "./App.css";
 import logo from "./logo.png";
-
+import { Routes, Route } from "react-router-dom";
+import Trash from "./Pages/Trash/Trash";
+import Profile from "./Pages/Profile/Profile";
+import Archieve from "./Pages/Archieve/Archieve";
+import Label from "./Pages/Labels/Labels";
+import Home from "./Pages/Home/Home";
+import Signup from "./Pages/Auth/Signup";
+import Login from "./Pages/Auth/Login";
+import Notes from "./Pages/Notes/Notes";
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Routes>
+        <Route path="/" element={<Home />}>
+          Home
+        </Route>
+        <Route path="/label" element={<Label />}>
+          Labels
+        </Route>
+        <Route path="/archieve" element={<Archieve />}>
+          Archieve
+        </Route>
+        <Route path="/trash" element={<Trash />}>
+          Trash
+        </Route>
+        <Route path="/profile" element={<Profile />}>
+          Profile
```

Comment:
is any reason you are passing text inside these route

Reply:
no that is by mistake